adminer-4.3.1-elasticsearch54.patch

```diff
@@ -83,17 +83,58 @@
  	}
  
  
-@@ -242,7 +293,7 @@ if (isset($_GET["elastic"])) {
+@@ -201,6 +252,8 @@ if (isset($_GET["elastic"])) {
+ 	}
+ 
+ 	function get_databases() {
++		$_GET['dbsize'] = 1;
++
+ 		global $connection;
+ 		$return = $connection->rootQuery('_aliases');
+ 		if ($return) {
+@@ -223,10 +276,17 @@ if (isset($_GET["elastic"])) {
+ 
+ 	function count_tables($databases) {
+ 		global $connection;
+-		$return = $connection->query('_mapping');
+-		if ($return) {
+-			$return = array_map('count', $return);
++		$return = array();
++		$result = $connection->query('_stats');
++
++		if (!(empty($result) || empty($result['indices']))) {
++			$indices = $result['indices'];
++			foreach($indices as $indice => $stats) {
++				$indexing = $stats['total']['indexing'];
++				$return[ $indice ] = $indexing['index_total'];
++			}
+ 		}
++
+ 		return $return;
+ 	}
+ 
+@@ -241,24 +301,32 @@ if (isset($_GET["elastic"])) {
+ 
  	function table_status($name = "", $fast = false) {
  		global $connection;
++<<<<<<< HEAD
```

Comment:
Ahhh... the joys of patching patches :-)   There's a stray merge conflict here.

Reply:
Sorry, there are new commits to fix and prevent it. I think you can setup travis to avoid this kind of things. There is a `.travis.yml` in this pull request that may fit our needs.